packages/libraries/core/src/client/http-client.ts

```diff
@@ -117,6 +120,8 @@ export async function makeFetchCall(
         body: config.body,
         headers: {
           'x-request-id': requestId,
+          'x-client-timestamp': new Date().toISOString(),
```

Comment:
You can use this maybe?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Date

Reply:
Decided to remove it for now after more discussion.